benefits/static/css/styles.css

```diff
@@ -121,11 +121,12 @@ a:visited:not(.btn) {
 a[target="_blank"]::after {
   content: " ";
   background-color: currentColor;
-
+  /* prettier-ignore */
   mask-image:
     url("/static/img/external-link.svg"),
     linear-gradient(transparent, transparent);
   mask-position: center center;
+  /* prettier-ignore */
```

Comment:
i'm not sure exactly what's going on but i'd guess the jockeying between prettier's auto-formatting in my containerized dev environment and what is applied in https://github.com/cal-itp/benefits/pull/3214/commits/4608a4b261fc8dedb564d4c674157084833ac109 is a side-effect of a version mismatch in the latest/greatest VS Code extension and the (deprecated) pre-commit mirror.

https://github.com/prettier/prettier/discussions/16227

happy to explore a solution that's more than a band-aid in a followup PR, but i didn't want to get sidetracked here.